salt/modules/dockerng.py

```diff
@@ -4168,174 +4304,13 @@ def signal_(name, signal):
 
 @_refresh_mine_cache
 @_ensure_exists
-def start(name, validate_ip_addrs=True, **kwargs):
+def start(name, **kwargs):
```

Comment:
We can drop the ``**kwargs`` here, since we are no longer doing anything with them.